doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -33,48 +35,104 @@ Generated keys default to retaining the seed used.
 The seed is also by default retained when keys are loaded from B<PKCS#8> files
 in the seed format.
 When available, the seed parameter is also used during key export and import,
-with keys regenerated from the seed even if separately provided on import.
+with keys regenerated from the seed even when also provided on import.
```

Comment:
I think this is wrong.  If the key is provided we should use it and ignore the seed's presence.

Why impose a performance penalty when both are present?  I see no reason for doing so.

Reply:
There is no significant performance penalty, the main cost is generating the matrix from `rho`.
This PR is addressing documentation, and adding a missing control, not changing the design.
Note that whether the seed is there in the first place is controlled by other parameters (retention keygen and preference on P8 import).
On export the seed is provided when known, some providers may need it, on import it is used when provided.
In order to not be used, we'd need to store the `provctx` with the key in "new", so that the `prefer_seed` property can be checked.
You're the first to suggest that the seed needs to be able to be ignored (when accompanied by an explicit key) on import between providers by a provider that is capable of using it. You can open a separate issue for that new idea.

Reply:
Issue is #26573 for OTC discussion.